tritium-metrics-jvm/src/main/java/com/palantir/tritium/metrics/jvm/JvmMetrics.java

```diff
@@ -216,6 +215,16 @@ protected RatioGauge.Ratio getRatio() {
         });
     }
 
+    private static void registerJvmBufferPools(TaggedMetricRegistry registry) {
+        JvmBufferMetrics jvmBufferMetrics = JvmBufferMetrics.of(registry);
+        for (BufferPoolMXBean pool : ManagementFactory.getPlatformMXBeans(BufferPoolMXBean.class)) {
```

Comment:
This is a change.

The [BufferPoolMetricSet](https://github.com/dropwizard/metrics/blob/release/4.2.x/metrics-jvm/src/main/java/com/codahale/metrics/jvm/BufferPoolMetricSet.java#L26) explicitly gets the `direct` and `mapped` pools. I think we can just report metrics for all the pools exposed via `ManagementFactory.getPlatformMXBeans(BufferPoolMXBean.class)`?

Reply:
The metric names are currently: `jvm.buffers.direct.<>` and `jvm.buffers.mapped.<>`. I'd like to maintain backward compatibility. Let's just get the metrics for those specific pools.